bibsonomy-rest-api-v2/docs/REMOVED_FEATURES.md

```
@@ -0,0 +1,770 @@
+
+**Decision**: Replace friend-based collaboration with group-based collaboration.
+
+### Codebase References
+
+#### Controllers (Webapp)
+- `/Users/tomvolker/localProjects/dsc/BibSonomy/bibsonomy-webapp/src/main/java/org/bibsonomy/webapp/controller/FriendPageController.java` - Single friend management
+- `/Users/tomvolker/localProjects/dsc/BibSonomy/bibsonomy-webapp/src/main/java/org/bibsonomy/webapp/controller/FriendsPageController.java` - Friends overview page (shows posts from friends)
+- `/Users/tomvolker/localProjects/dsc/BibSonomy/bibsonomy-webapp/src/main/java/org/bibsonomy/webapp/controller/FriendsOverviewController.java` - Friends list management
+- `/Users/tomvolker/localProjects/dsc/BibSonomy/bibsonomy-webapp/src/main/java/org/bibsonomy/webapp/controller/SpheresPageController.java` - Friend groups/"spheres" management
+
+#### Commands (Webapp)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)